Gemfile

```diff
@@ -6,10 +6,11 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby "2.6.4"
 
 gem "active_link_to", "~> 1.0.5"
+gem "active_storage_svg_sanitizer", "~> 0.1.0"
 gem "acts_as_commentable_with_threading", "~> 2.0.1"
 gem "addressable", "2.6.0"
 gem "airbrake", "~> 9.3"
-gem "aws-sdk-s3", "~> 1.46.0", require: false
+gem "aws-sdk-s3", "~> 1.48.0", require: false
```

Comment:
Does it require gem update? If it's not part of it, why not make a separate PR for it?

Reply:
Mostly because we're an extremely small shop and we're also lazy.